frontend/pages/sign-in/code.tsx

```diff
@@ -0,0 +1,217 @@
+import { ChangeEvent, ClipboardEvent, useCallback, useEffect } from 'react';
+
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import { useRouter } from 'next/router';
+
+import { withLocalizedRequests } from 'hoc/locale';
+
+import useMe from 'hooks/me';
+
+import { loadI18nMessages } from 'helpers/i18n';
+
+import Alert from 'components/alert';
+import Button from 'components/button';
+import ErrorMessage from 'components/forms/error-message';
+import Input from 'components/forms/input';
+import Loading from 'components/loading';
+import { Paths, UserRoles } from 'enums';
+import NakedLayout, { NakedLayoutProps } from 'layouts/naked';
+import { PageComponent } from 'types';
+import { SignIn, SignInCodeForm } from 'types/sign-in';
+
+import { useSignIn } from 'services/authentication/authService';
+import { useAcceptInvitation, useInvitedUser } from 'services/invitation/invitationService';
+
+export const getServerSideProps = withLocalizedRequests(async (context) => {
+  // Property 'query' does not exist on type 'GetStaticPropsContext<ParsedUrlQuery, PreviewData>'
+  /** @ts-ignore */
```

Comment:
`withLocalizedRequests` accepts a generic type:

```suggestion
export const getServerSideProps = withLocalizedRequests<GetServerSideProps>(async (context) => {
```